jest.config.ts

```diff
@@ -10,7 +12,9 @@ module.exports = {
   ],
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  testPathIgnorePatterns: ["/node_modules/", "tests-e2e"],
+  testPathIgnorePatterns: ["/node_modules/", "tests-e2e/(?!jest)"],
```

Comment:
Slightly awkward: `tests-e2e` now has:
- playwright tests, which are really e2e
- a few jest tests which build with hugo + assert build failed as expected

The jest tests share enough in common with the playwright tests (namely, building via hugo) I think it makes sense to keep them there.